lib/diagnostic.rb

```diff
@@ -6,21 +6,23 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'print or puts'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = 'pry'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = "movie: 'star wars episode viii'"
```

Comment:
This will cause an error.  We just want the name of the variable based on the given string.  Ruby's naming convention is "snake case" so this should just be `star_wars_episode_viii`.